src/cares_wrap.cc

```diff
@@ -15,18 +15,16 @@
 #include <stdlib.h>
 #include <string.h>
 
-#if defined(__ANDROID__) || \
-    defined(__MINGW32__) || \
-    defined(__OpenBSD__) || \
+#if defined(__ANDROID__) || defined(__MINGW32__) || defined(__OpenBSD__) ||    \
     defined(_MSC_VER)
 
-# include <nameser.h>
```

Comment:
IMHO indenting here makes things more readable, just like you'd indent for a normal conditional block outside of macros.